ocs_ci/utility/utils.py

```diff
@@ -1997,6 +1997,34 @@ def destroy_cluster(installer, cluster_path, log_level="DEBUG"):
         log.error(traceback.format_exc())
 
 
+def config_to_string(config):
+    """
+    Convert ConfigParser object to string in INI format.
+
+    Args:
+        config (obj): ConfigParser object
+
+    Returns:
+        str: Config in one string
+
+    """
+    string = ""
```

Comment:
i would use meaningful name like "cfg_str" or something else that can imply about string purpose